db/migrations/20170326215143_initial.js

```diff
@@ -1,30 +1,74 @@
 
 exports.up = function (knex, Promise) {
   return Promise.all([
-    knex.schema.createTableIfNotExists('profiles', function (table) {
+    // knex.schema.createTableIfNotExists('profiles', function (table) {
+    //   table.increments('id').unsigned().primary();
+    //   table.string('first', 100).nullable();
+    //   table.string('last', 100).nullable();
+    //   table.string('display', 100).nullable();
+    //   table.string('email', 100).nullable().unique();
+    //   table.string('phone', 100).nullable();
+    //   table.timestamps(true, true);
+    // }),
+    // knex.schema.createTableIfNotExists('auths', function(table) {
+    //   table.increments('id').unsigned().primary();
+    //   table.string('type', 8).notNullable();
+    //   table.string('oauth_id', 30).nullable();
+    //   table.string('password', 100).nullable();
+    //   table.string('salt', 100).nullable();
+    //   table.integer('profile_id').references('profiles.id').onDelete('CASCADE');
+    // })
+    knex.schema.createTableIfNotExists('users', function(table) {
       table.increments('id').unsigned().primary();
-      table.string('first', 100).nullable();
-      table.string('last', 100).nullable();
-      table.string('display', 100).nullable();
-      table.string('email', 100).nullable().unique();
-      table.string('phone', 100).nullable();
-      table.timestamps(true, true);
+      table.string('email', 100).nullable();
+      table.string('github_handle', 100).notNullable();
+      table.string('profile_photo', 200).nullable();
     }),
-    knex.schema.createTableIfNotExists('auths', function(table) {
+    knex.schema.createTableIfNotExists('boards', function(table) {
       table.increments('id').unsigned().primary();
-      table.string('type', 8).notNullable();
-      table.string('oauth_id', 30).nullable();
-      table.string('password', 100).nullable();
-      table.string('salt', 100).nullable();
-      table.integer('profile_id').references('profiles.id').onDelete('CASCADE');
+      table.string('board_name', 50).notNullable();
+      table.string('repo_name', 100).nullable();
+      table.string('repo_url', 200).nullable();
+      table.integer('owner_id').references('users.id').onDelete('CASCADE');

```

Comment:
1) Why are you creating a direct reference between these two tables? 
2) If you are directly creating a reference to the users table what is the point of the join table?